src/main/java/org/killbill/billing/plugin/adyen/api/ExpiredPaymentPolicy.java

```diff
@@ -44,7 +46,7 @@ public boolean isExpired(final List<PaymentTransactionInfoPlugin> paymentTransac
             return false;
         }
 
-        final PaymentTransactionInfoPlugin transaction = latestTransaction(paymentTransactions);
+        final AdyenPaymentTransactionInfoPlugin transaction = (AdyenPaymentTransactionInfoPlugin) latestTransaction(paymentTransactions);
```

Comment:
I've tried doing that in the `isExpired` method, but there's no clean way to cast from `List<PaymentTransactionInfoPlugin>`, and that's what we `super.getPaymentInfo(kbAccountId, kbPaymentId, properties, context)`